src/server/utilities/cspHeader/index.test.js

```diff
@@ -228,6 +230,7 @@ describe('cspHeader', () => {
         'https://www.instagram.com',
         'https://cdn.syndication.twimg.com',
         'https://public.flourish.studio',
+        'https://client.rum.us-east-1.amazonaws.com',
```

Comment:
What's the reason for including the US EC2 in the CSP Header?

Reply:
This is where the AWS cloudrum script is hosted - https://github.com/bbc/simorgh/pull/10229/files/8e03f0aa83e14af3e0088c93b76386b64d0ded0f#diff-ddd236da434bde238fa4a12aa7c954e90ae577538c305aa4e9f15de9694257d4R7

Content security policy is like an allow-list - it tells browsers it's okay to download stuff from this domain

It's using a US domain just because AWS only seem to make their script available through that URL - they don't offer a european URL